### PR TITLE
Add instructions to add new homework

### PR DESCRIPTION
**Pull request by gmuraru**
Instructions to add a new homework for a specific course.

Fixes https://github.com/vmck/acs-interface/issues/125

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -111,3 +111,20 @@ To stop, hit `CTRL-C` in the terminal where you started `runserver`, and
 The deployment is orchestrated at https://github.com/vmck/acs-deploy - it's the
 living documentation and single instance of how to run this thing in
 production.
+
+## Add new homework
+1. You need to have access to the [admin page for V2 vmchecker](http://v2.vmchecker.cs.pub.ro/admin).
```
Comment (mgax) [inline Readme.md]: Please write the address with `https://`. The `http://` does indeed redirect to the https version, but it's more secure if you link directly to https.

Comment (gmuraru) [reply Readme.md]: Done

File: Readme.md [hunk 2]
```diff
@@ -111,3 +111,20 @@ To stop, hit `CTRL-C` in the terminal where you started `runserver`, and
 The deployment is orchestrated at https://github.com/vmck/acs-deploy - it's the
 living documentation and single instance of how to run this thing in
 production.
+
+## Add new homework
+1. You need to have access to the [admin page for V2 vmchecker](https://v2.vmchecker.cs.pub.ro/admin).
+2. Click on adding a new *Assignment*
+  - Select the course (mandatory field);
+  - Add a code for the homework (currently it is only displayed in the admin page);
```
Comment (Mr-Vinti) [inline Readme.md]: homework code also shows up in Results page link and list all submissions page

Comment (mgax) [reply Readme.md]: I've changed the wording per @Mr-Vinti's suggestion - https://github.com/vmck/acs-interface/commit/85cf502103407f72833a8933075fab285e679d97

**Pull request merged**